iroha_data_model/src/isi.rs

```diff
@@ -141,12 +141,12 @@ where
 
 /// Generic instruction for a removal of an object from the identifiable destination.
 #[derive(Debug, Clone, Serialize, Deserialize, Encode, Decode)]
-pub struct Remove<D, O>
+pub struct Substract<D, O>
```

Comment:
```suggestion
pub struct Subtract<D, O>
```

Reply:
@lebdron - thanks, regex replacement was not a great idea)))